### PR TITLE
Add an AppStream metadata file

### LINKED ISSUE #191: AppStream information not available
https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html

It would make it easier to find the application on software stores.
You can find plenty of examples in KDE and elsewhere. ;)

### PR DESCRIPTION
**Pull request by Calinou**
This closes #191.

PS: I added the email of the most active maintainer as the `update_contact`, let me know if this should be changed: slightly_smiling_face:

### CODE REVIEW

File: com.kdab.Hotspot.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <url type="homepage">https://github.com/KDAB/hotspot</url>
+  <url type="bugtracker">https://github.com/KDAB/hotspot/issues</url>
+
+  <releases>
+    <release version="1.2.0" date="2019-05-20"/>
+  </releases>
+
+  <content_rating type="oars-1.1" />
+  <developer_name>KDAB</developer_name>
+  <update_contact>milian.wolff_AT_kdab.com</update_contact>
```
Comment (milianw) [inline com.kdab.Hotspot.appdata.xml]: is the _AT_ standard procedure? my mail is public, so you could use @ directly

**Comment by milianw** [comment]
one question, otherwise lgtm - thanks!

File: com.kdab.Hotspot.appdata.xml (hunk 1 above)
Comment (Calinou) [reply com.kdab.Hotspot.appdata.xml]: The [AppStream metadata guidelines](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html) state that using `_AT_` is valid, but I'll replace it with `@` if that's OK with you :slightly_smiling_face:

**Comment by milianw** [comment]
thanks!

**Pull request merged**